### PR TITLE
Custom 404 maken

### LINKED ISSUE #97: Custom 404 maken
Branch name: `feature/custom-404-maken`

**Omschrijving:**
Staat iets netter. Later kunnen we dit nog verbeteren/leuker maken, maar voor nu is het de bedoeling om de basis te doen.

### PR DESCRIPTION
**Pull request by jochemvogel**
# Description

Closes #97 

Branch name: `feature/custom-404-maken`

Lees ticket voor omschrijving

## Type of change

Please select the option that is relevant

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

# Checklist

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] All the required documentation is added.
- [ ] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.

# Additional information

_[Provided extra information if necessary]_

### CODE REVIEW

File: src/routes/__error.svelte [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	function sendMail() {}
+</script>
+
+{#if status === 404}
+	<h1>Pagina bestaat niet</h1>
+	<p>De pagina die jij probeert te bereiken, bestaat niet.</p>
+	<p>
+		Probeer het nogmaals en neem anders contact op met de administrator. Deze gaat het proberen op
+		te lossen.
+	</p>
```
Comment (RooyyDoe) [inline src/routes/__error.svelte:26]: misschien kan je deze nog naar het midden centreren?

Comment (jochemvogel) [reply src/routes/__error.svelte:26]: Jaa, was even snel gedaan. Goed om hier de komende week even rustig naar te kijken. Wilde het nu snel doen, maar eerst even rustig een design maken, is wat beter denk ik.

**Comment by jochemvogel** [comment]
image

**Comment by jochemvogel** [comment]
Later meer tijd besteden aan alle error pagina's (ook offline), maar voor nu is dit prima.

**Pull request closed without merging**